packages/babel-preset-env/src/utils.js

```diff
@@ -77,56 +93,62 @@ export const prettifyTargets = (targets: Targets): Object => {
     results[target] = value;
     return results;
   }, {});
-};
+}
 
-export const isUnreleasedVersion = (version: string, env: string): boolean => {
+export function isUnreleasedVersion(version, env) {
   const unreleasedLabel = unreleasedLabels[env];
   return (
     !!unreleasedLabel && unreleasedLabel === version.toString().toLowerCase()
   );
-};
+}
 
-export const getLowestUnreleased = (
-  a: string,
-  b: string,
-  env: string,
-): string => {
+export function getLowestUnreleased(a, b, env) {
   const unreleasedLabel = unreleasedLabels[env];
   const hasUnreleased = [a, b].some(item => item === unreleasedLabel);
   if (hasUnreleased) {
     return a === hasUnreleased ? b : a || b;
   }
   return semverMin(a, b);
-};
+}
 
-export const filterStageFromList = (list: any, stageList: any) => {
+export function filterStageFromList(list, stageList) {
```

Comment:
We should keep the flow types. What is the reason you removed all of them?

Reply:
@danez  see https://github.com/babel/babel/pull/7646#discussion_r257186983

Reply:
I saw that, but disabling a file is something different than removing all existent type declarations. I'm fine to disable a file for now, but removing all types seems unnecessary.

Reply:
We should check if now that everything is finished flow types can be re-introduced without type warnings.
@zloirock If you prefer, I can do that.

Reply:
I can also help with restoring flow types, if needed!

Reply:
@nicolo-ribaudo that could be great -)